plugins/github/current_github.mli

```diff
@@ -1,6 +1,7 @@
 (** Integration with GitHub. *)
 
 val webhook : engine:Current.Engine.t
+              -> get_job_ids:(owner:string -> name:string -> hash:string -> string list)
```

Comment:
Can you add an explanation of this and how to use it in the module documentation?